packages/consoledot-api/src/queries/useUser.ts

```diff
@@ -20,7 +31,10 @@ export function useUser({getUser, suspense}:{getUser:() => Promise<{username: st
       }
 
       const canOpenConnection = (instance: KafkaInstanceEnhanced): boolean => {
-        return instance ? ReadyStatuses.includes(instance.status) : false;
+        return instance
+          ? !SuspendedStatuses.includes(instance.status) ||
+              !DeletingStatuses.includes(instance.status)
```

Comment:
I think you meant that or to be an and?

Or this can be simplified to

```suggestion
          ? ![...SuspendedStatuses, ...DeletingStatuses].includes(instance.status)
```